release_notes/ocp-4-14-release-notes.adoc

```diff
@@ -2901,11 +2925,11 @@ For any {product-title} release, always review the instructions on xref:../updat
 
 //Update with relevant advisory information
 [id="ocp-4-14-0-ga"]
-=== RHSA-2023:1234 - {product-title} 4.14.0 image release, bug fix, and security update advisory
+=== RHSA-2023:1234 - {product-title} {product-version}.0 image release, bug fix, and security update advisory
```

Comment:
I think we have the ART Errata now? Can probably update these, too.

Reply:
Good catch, Shauna. Updated to match the advisory numbers on https://issues.redhat.com/browse/ART-7691.

@JoeAldinger . A further update is due for this section based on the completion of https://issues.redhat.com/browse/OSDOCS-8362 ?